doc/whats_new/v1.3.rst

```diff
@@ -479,6 +479,13 @@ Changelog
   accepts sparse metrics.
   :pr:`19664` by :user:`Kaushik Amar Das <cozek>`.
 
+:mod:`sklearn.multiclass`
+..............................
+
+- |Enhancement| Raise informative warning message in :func:`type_of_target` when
+  represented as bytes.
```

Comment:
This is a API change because it requires users to update their code to remove the warning:

```suggestion
- |API| Raise informative warning message in :func:`type_of_target` when
  represented as bytes. For classifiers and classification metrics, labels encoded 
  as bytes is deprecated and will raise an error in v1.5.
```